src/wrapper/qr_mumps_common.jl

```diff
@@ -0,0 +1,65 @@
+struct qrm_spmat{T}  # T ∈ Union{Complex{Float32}, Complex{Float64}, Float32, Float64}
+  irn :: Ptr{Cint}
+  jcn :: Ptr{Cint}
+  val :: Ptr{T}
+  m   :: Cint
+  n   :: Cint
+  nz  :: Cint
+  sym :: Cint
+  h   :: Ptr{Cvoid}
+end
```

Comment:
`qmr_spmat{T}` is the matrix that we want to factorize. We pass it to `qr_mumps` in a COO format and `qr_mumps` stores the problem with its own storage, specialized for the multifrontal decomposition. `h` is a pointer to this model.

Because the type of `val` is `Vector{T}`, we can easily dispatch between `sqrm`, `cqrm`, `zqrm`, `dqrm` routines.